autogpt_platform/frontend/src/app/(platform)/profile/(user)/dashboard/components/AgentTableRow/AgentTableRow.tsx

```diff
@@ -90,7 +90,10 @@ export const AgentTableRow = ({
             />
           </div>
           <div className="flex flex-col">
-            <h3 className="text-[15px] font-medium text-neutral-800 dark:text-neutral-200">
+            <h3
+              className="text-[15px] font-medium text-neutral-800 dark:text-neutral-200"
```

Comment:
I thought we decided to do even sizing? Does text-sm differ a lot, or can we bear with it?